File: 프로그래머스/lv2/12985. 예상 대진표/예상 대진표.js
function solution(n,a,b){
    
    let cnt = 0;
    while(true){
        cnt++;
        if(Math.ceil(a/2) === Math.ceil(b/2)){
            return cnt;
        }
        a = Math.ceil(a/2);
        b = Math.ceil(b/2)
    }
}

File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.java
class Solution
{
    public int solution(int n, int a, int b)
    {
        int cnt = 0;
        while (true) {
            cnt ++;
            int A = (int) Math.ceil((double) a / 2);
            int B = (int) Math.ceil((double) b / 2);
            if (A == B) {
                return cnt;
            }
            a = A;
            b = B;
        }
    }
}